openvpn-wire-protocol.xml

```diff
@@ -1390,10 +1390,10 @@ available) to drop revoked keys.)</li>
         must be split over multiple control messages.
         </t>
         <t>
-          Control message should be also separated by a NUL byte. The current implementation might not
+          Control messages should be also separated by a NULL byte. The current implementation might not
           enforce this behaviour but future implementation that no longer rely on TLS framing might enforce
           this behaviour. When sending an implementation MUST send each control message in its own TLS
-          record terminated by a NUL byte.
+          record terminated by a NULL byte.
```

Comment:
RFC typically refer to the ACSII byte with value 0 as `NUL`. `NULL` is typically for pointers